src/main/java/io/numaproj/numaflow/sinker/Response.java

```diff
@@ -13,9 +13,9 @@
 @AllArgsConstructor(access = AccessLevel.PRIVATE)
 public class Response {
   private final String id;
-  private final boolean success;
+  private final Boolean success;
```

Comment:
Change it back because it's backward incompatible. If we ever decide to have a backward in-compatible release, we can make this change. Tracking it [here](https://github.com/numaproj/numaflow-java/issues/185).